packages/xo-server/src/xo-mixins/xen-servers.mjs

```diff
@@ -371,6 +386,37 @@ export default class {
           return xapiObjectToXo(object, dependents)
         }
 
+        const markPool = async () => {
+          const now = Date.now()
+          const { pool } = xapi
+
+          try {
+            await asyncEach(Object.entries(pool.other_config), ([key, value]) => {
+              if (key.startsWith(poolMarkingPrefix)) {
+                const { lastConnected } = JSON.parse(value)
+                if (now - lastConnected > poolMarkingMaxAge) {
+                  return pool.update_other_config(key, null)
+                }
+              }
+            })
+
+            const publicUrl = config.getOptional('http.publicUrl')
+            const info = {
+              lastConnected: now,
+              publicUrl,
+            }
+            if (publicUrl === undefined) {
+              info.networkInterfaces = networkInterfaces()
```

Comment:
I don't remember what we said about also adding `xo-server`'s port.

Reply:
I don't think we decided on anything specific, IMHO, it can get quite complicated, I think we should avoid these and use HTTPs by default, `http.publicUrl` can be used for special cases.